org/elmot/index.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: org
+title: Elijah Motornyy(http://elmot.xyz)
```

Comment:
You should leave the URL out of the title; it'll show up as a prefix to all PIDs under this org.